.github/workflows/detect-git-lfs.yml

```diff
@@ -17,5 +17,5 @@ jobs:
       run: |
         git diff -U0 --ignore-all-space $(git merge-base origin/${{ github.event.pull_request.base.ref }} ${{ github.event.pull_request.head.sha }}) \
         | grep "^oid sha236" \
-        || ( echo "No pointers found!" ; exit 0 ) \
-        && ( echo "Git LFS pointers found; not allowed to add LFS files to Trilinos!" ; exit 1)
+        || { echo "No pointers found!" ; exit 0 } \
+        && { echo "Git LFS pointers found; not allowed to add LFS files to Trilinos!" ; exit 1 
```

Comment:
This should be a script that takes arguments.  Then this could be tested and run locally.

Reply:
I can do that.  Yes I did test the overall logic locally.  However, I think I want it to march through all of the intermediary commits and run this on each one.  Will make a script and get back.